app/uk/gov/hmrc/incorporatedentityidentification/controllers/RegisterBusinessEntityController.scala

```diff
@@ -35,8 +35,8 @@ class RegisterBusinessEntityController @Inject()(cc: ControllerComponents,
   def registerLimitedCompany(): Action[JsValue] = Action.async(parse.json) {
     implicit request =>
       authorised() {
-        val crn = (request.body \"crn").as[String]
-        val ctutr = (request.body  \"ctutr").as[String]
+        val crn = (request.body \ "crn").as[String]
+        val ctutr = (request.body \ "ctutr").as[String]
```

Comment:
You need to read the regime that the FE is passing down here. I think you did this in your previous commit